packages/qwik/src/core/v2/shared/vnode-data-types.ts

```diff
@@ -34,12 +34,12 @@ export const VNodeDataSeparator = {
   ADVANCE_512: /* ******* */ 42, // `*` is vNodeData separator skipping 256.
   ADVANCE_1024_CH: /* ** */ `+`, // `+` is vNodeData separator skipping 512.
   ADVANCE_1024: /* ****** */ 43, // `+` is vNodeData separator skipping 512.
-  ADVANCE_2048_CH: /* *  */ '`', // '`'` is vNodeData separator skipping 1024.
-  ADVANCE_2048: /* ****** */ 44, // '`'` is vNodeData separator skipping 1024.
+  ADVANCE_2048_CH: /* *  */ ',', // ',' is vNodeData separator skipping 1024.
+  ADVANCE_2048: /* ****** */ 44, // ',' is vNodeData separator skipping 1024.
   ADVANCE_4096_CH: /* *  */ `.`, // `.` is vNodeData separator skipping 2048.
   ADVANCE_4096: /* ****** */ 46, // `.` is vNodeData separator skipping 2048.
```

Comment:
I don't know why we are missing the `45` charcode?

Reply:
That is weird! I don't think we should be skipping anything. I think that is a bug and should be fixed.